### PR TITLE
Release/1.1.0

### PR DESCRIPTION
**Pull request by matus-tomlein**
**CHANGELOG**

Update mobile tracker dependencies to v3 (#141)
Add deep link and message notification events (#142)
Update demo app to show accessing tracker instance from platform native code (#136)
Fix check for byteLimitGet when reading emitter configuration (#138)

**Under the hood**

Fix compilation error for iOS build on ARM macs (#140)
Fix check for Android version in GitHub Actions (#134) 
Update year in all copyright notices (#148) 
Update dependencies (#151)

# Documentation

* [PR for deep link and message notification events](https://github.com/snowplow-incubator/data-value-resources/pull/43).
* [PR for accessing native trackers](https://github.com/snowplow-incubator/data-value-resources/pull/37) – already reviewed.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,15 @@
+Version 1.1.0 (2022-02-07)
+--------------------------
+Update mobile tracker dependencies to v3 (#141)
+Add deep link event and context (#143)
+Add MessageNotification event (#142)
```
Comment (adatzer) [inline CHANGELOG]: There is one commit that closes both issues (142,143). Not an objection, just mentioning in case this was not intentional.

**Review by adatzer — APPROVED** [review]
Looks great!

File: CHANGELOG (hunk 1 above)
Comment (colmsnowplow) [reply CHANGELOG]: Can it be split into two commits, or can the issues be combined into one? The rule of thumb we follow is a 1:1:1 map between commit:issue:change made

Comment (matus-tomlein) [reply CHANGELOG]: That's true, I messed up when creating the same PR for both issues. I merged the issues into one and renamed the commit to refer to the issue. That seemed easier at this point and the issues are quite related.

**Pull request merged**